packages/typegpu/src/experimental/index.ts

```diff
@@ -48,6 +49,12 @@ export const tgpu = {
   read,
   write,
 };
+
+// this method is used by the vite plugin and should not be visible to the user
```

Comment:
Tiny nitpick, more general 'rollup' instead of 'vite'
```suggestion
// Hidden API, used only by tooling (e.g., rollup plugin).
```